v3/fleet-local/it-hybrid/memcached.service

```diff
@@ -10,7 +10,7 @@ Restart=always
 RestartSec=5
 TimeoutStartSec=0
 
-ExecStartPre=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && docker pull borja/docker-memcached"
+ExecStartPre=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && docker pull index.docker.io/borja/docker-memcached"
```

Comment:
should we also lock to a version? seems like a silly thing to allow to be unreliable